### PR TITLE
Optim visualization

### PR DESCRIPTION
**Pull request by AntoineCAMILLE**

### CODE REVIEW

File: public/js/jobManager.js [hunk 1]
```diff
@@ -14,6 +14,10 @@ function buildDownloadLink(jobId, state) {
     + '" href="' + url + '">' + msg + '</a>';
 }
 
+function buildResultLink(jobId) {
+  return '<a href="/result.html?api_key=' + getParams()['api_key'] + '&job_id=' + jobId + '">Visualiser les résultats</a>'
```
Comment (fonsecadeline) [inline public/js/jobManager.js:18]: Could this open a new page rather than changing current one ?

Comment (AntoineCAMILLE) [reply public/js/jobManager.js:18]: Actually i'm in school so i can't change it until tomorrow
But you can 'middle click' on the link to open it on a new page

Comment (fonsecadeline) [reply public/js/jobManager.js:18]: Yes, I was talking about changing default behaviour :)

[the remark this answers is not in the document]
Comment (AntoineCAMILLE) [reply public/js/jobManager.js:18]: Actually there's no i18n implemented

[the remark this answers is not in the document]
Comment (fab-girard) [reply public/js/jobManager.js:18]: yes there is i18n in this project: https://github.com/Mapotempo/optimizer-api/blob/master/public/js/conf.js

[the remark this answers is not in the document]
Comment (AntoineCAMILLE) [reply public/js/jobManager.js:18]: This is not an I18n, there is no locales, or the only one is the french one
So we have to implement a REAL I18n to do what you except

[the remark this answers is not in the document]
Comment (fab-girard) [reply public/js/jobManager.js]: I had a dream...

[the remark this answers is not in the document]
Comment (fab-girard) [reply public/js/jobManager.js]: I had a dream...

**Pull request merged**